source/containers/FormField/FormField.tsx

```diff
@@ -326,17 +331,20 @@ const FormField = (props: FormFieldProps): JSX.Element => {
   return (
     <View>
       {label ? (
-        <Label
-          colorSchema={validColorSchema}
-          underline={labelLine}
-          help={
-            !input.helpInComponent && help && Object.keys(help).length > 0
-              ? help
-              : {}
-          }
-        >
-          {label}
-        </Label>
+        <>
+          <Label
+            colorSchema={validColorSchema}
+            underline={labelLine}
+            help={
+              !input.helpInComponent && help && Object.keys(help).length > 0
+                ? help
+                : {}
+            }
+          >
+            {label}
+          </Label>
+          <Spacer />
+        </>
```

Comment:
It is a little weird using a Text component for spacing, how about making a label container instead?
```js

const LabelContainer = styled.View`
width: 100%;
padding-bottom: 16px;
`;

<LabelContainer>
          <Label
            colorSchema={validColorSchema}
            underline={labelLine}
            help={
              !input.helpInComponent && help && Object.keys(help).length > 0
                ? help
                : {}
            }
          >
            {label}
          </Label>
        </LabelContainer>
```